libraries/EEPROM/src/EEPROM.h

```diff
@@ -117,11 +105,10 @@ class EEPROMClass {
     template <class T> T writeAll (int address, const T &);
 
   protected:
-    uint32_t _sector;
+    nvs_handle _handle;
```

Comment:
could you maybe forward declare the `nvs_handle` so it is not required to include nvs in the header?